### DISCUSSION TITLE
20230723報告問題檢討修正

### DISCUSSION THREAD

**Discussion by ChiuAlfredo** [ChiuAlfredo/options_day_data#42] [General]
把有問題的地方，列上來也可以同時想要怎麼處理

**Comment by Will150143** [comment]
1.圖表的縱橫軸的尺標要一致 → 程式多加軸的限制
2.圖表旁通常要配觀察，跟文字不搭會讓聽眾困惑 → 把觀察跟延伸的問題一起放上簡報
3.問句或肯定句要確定，標點符號要用對 → 自己練習報告時要注意這一句是問句還是肯定句
4.要搭配其他時間的資料驗證此時的結果是否正常 → 就多做吧或讀文章
5.說明樣本數時要說明最多會多少以及為何有差別 → 一樣放上簡報
6.公式要注意斜體 → 查詢其他文章怎麼呈現
7.簡報要做到一目了然 → 把自己當作第一次聽

**Reply by TsuYiii** [reply]
4.結算日當天不同頻率，例如：5分鐘、10分鐘、1小時等等

**Comment by TsuYiii** [comment]
1.針對結算日有相對大量的交易＝>討論交易量是否是一個能夠探討買權與賣權的報酬率的考量因子
2.研究標題與內容的差異度過大＝>針對標題進行修改，以及核對後面的研究方式能否對應到更正後的研究目的
3.講的內容不夠肯定，會被challenge=>預先思考可能被問的方向，進行引導，確認提問者提問的是我能夠回答的

**Reply by TsuYiii** [reply]
討論無自然對數的數值

過濾大的數值
討論爆量

**Comment by ChiuAlfredo** [comment]
1. 公式斜體沒有注意到
2. 沒有再進一步討論到特殊值得問題
